### PR TITLE
Fix to bulk_update_metadata spec.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
So that spec passes correctly.

## How was this change tested?
Tested against stage master.

## Which documentation and/or configurations were updated?
No

### CODE REVIEW

File: spec/features/bulk_update_metadata_spec.rb [hunk 1]
```diff
@@ -43,8 +43,11 @@
       next unless tds[3].text == note
 
       tds[9].find('form > button').click
-      # Confirm delete in the popup
-      click_link 'Delete'
+      within('#confirm-delete-modal') do
```
Comment (jcoyne) [inline spec/features/bulk_update_metadata_spec.rb:46]: Can you explain why we need this qualifier?

Comment (justinlittman) [reply spec/features/bulk_update_metadata_spec.rb:46]: It didn't work before; it works now. I can't offer a satisfying explanation.

Wondering if it has something to do with the recent changes to modals?

**Comment by jcoyne** [comment]
@justinlittman I was able to run spec/features/bulk_update_metadata_spec.rb  without this change and it passes.

**Comment by justinlittman** [comment]
@jcoyne With argo master deployed?

**Comment by jcoyne** [comment]
@justinlittman yes.

**Comment by mjgiarlo** [comment]
FWIW, this test is passing for me w/o this change.

File: spec/features/bulk_update_metadata_spec.rb (hunk 1 above)
Comment (peetucket) [reply spec/features/bulk_update_metadata_spec.rb:46]: possibly another delete button was added elsewhere on the page and the simple 'Delete' selector became ambiguous?

Comment (jcoyne) [reply spec/features/bulk_update_metadata_spec.rb:46]: @peetucket Capybara doesn't allow ambiguous selectors.

**Comment by mjgiarlo** [comment]
@justinlittman can you try running what's in `master` now (with ffx, not chrome)?

**Comment by justinlittman** [comment]
Master failed. Trying this branch.

**Comment by justinlittman** [comment]
Both failed.

**Comment by justinlittman** [comment]
Oh, if I sit and stare at it, master passes.

**Pull request closed without merging**